tests/tracepoint_format_parser.cpp

```diff
@@ -17,134 +19,140 @@ class MockTracepointFormatParser : public TracepointFormatParser
 TEST(tracepoint_format_parser, tracepoint_struct)
 {
   std::string input =
-    "name: sys_enter_read\n"
-    "ID: 650\n"
-    "format:\n"
-    "	field:unsigned short common_type;	offset:0;	size:2;	signed:0;\n"
-    "	field:unsigned char common_flags;	offset:2;	size:1;	signed:0;\n"
-    "	field:unsigned char common_preempt_count;	offset:3;	size:1;	signed:0;\n"
-    "	field:int common_pid;	offset:4;	size:4;	signed:1;\n"
-    "\n"
-    "	field:int __syscall_nr;	offset:8;	size:4;	signed:1;\n"
-    "	field:unsigned int fd;	offset:16;	size:8;	signed:0;\n"
-    "	field:char * buf;	offset:24;	size:8;	signed:0;\n"
-    "	field:size_t count;	offset:32;	size:8;	signed:0;\n"
-    "\n"
-    "print fmt: \"fd: 0x%08lx, buf: 0x%08lx, count: 0x%08lx\", ((unsigned long)(REC->fd)), ((unsigned long)(REC->buf)), ((unsigned long)(REC->count))\n";
-
-  std::string expected =
-    "struct _tracepoint_syscalls_sys_enter_read\n"
-    "{\n"
-    "  unsigned short common_type;\n"
-    "  unsigned char common_flags;\n"
-    "  unsigned char common_preempt_count;\n"
-    "  int common_pid;\n"
-    "  int __syscall_nr;\n"
-    "  u64 fd;\n"
-    "  char * buf;\n"
-    "  size_t count;\n"
-    "};\n";
+      "name: sys_enter_read\n"
+      "ID: 650\n"
+      "format:\n"
+      "	field:unsigned short common_type;	offset:0;	size:2;	"
+      "signed:0;\n"
+      "	field:unsigned char common_flags;	offset:2;	size:1;	"
+      "signed:0;\n"
+      "	field:unsigned char common_preempt_count;	offset:3;	"
+      "size:1;	signed:0;\n"
+      "	field:int common_pid;	offset:4;	size:4;	signed:1;\n"
+      "\n"
+      "	field:int __syscall_nr;	offset:8;	size:4;	signed:1;\n"
+      "	field:unsigned int fd;	offset:16;	size:8;	signed:0;\n"
+      "	field:char * buf;	offset:24;	size:8;	signed:0;\n"
+      "	field:size_t count;	offset:32;	size:8;	signed:0;\n"
+      "\n"
+      "print fmt: \"fd: 0x%08lx, buf: 0x%08lx, count: 0x%08lx\", ((unsigned "
+      "long)(REC->fd)), ((unsigned long)(REC->buf)), ((unsigned "
+      "long)(REC->count))\n";
```

Comment:
Clang-format is really annoying sometimes... could you disable it for this input string please? I think it'd be easier to read if we keep the original formatting.
See here for how to disable clang-format: https://github.com/iovisor/bpftrace/blob/471e79b76ab380bad559ab5f162204a041fc3146/src/arch/x86_64.cpp#L9
https://github.com/iovisor/bpftrace/blob/471e79b76ab380bad559ab5f162204a041fc3146/src/arch/x86_64.cpp#L48